src/cljs/main/broadfcui/page/workspace/monitor/tab.cljs

```diff
@@ -35,7 +35,9 @@
        :render #(links/create-internal {:data-test-id (str "submission-" %)
                                         :href (nav/get-link :workspace-submission workspace-id %)}
                   "View")}
-      {:header "Status" :as-text :status :sort-by :text
+      {:header "Status"
+       :sort-by (fn [submission] (:status submission))
+       :as-text (fn [submission] (.stringify js/JSON (clj->js (:workflowStatuses submission))))
```

Comment:
Dunno how users will respond to JSON - but it works great as a first iteration.  👍